pkg/autoscaler/autoscaler_server.go

```diff
@@ -129,22 +131,41 @@ func (s *AutoScaler) pollAPIServer() {
 			r := resource.NewQuantity(0, guessFormat(res))
 			r.SetMilli(want)
 			newReqs[ctr].Requests[apiv1.ResourceName(res)] = *r
-			glog.V(0).Infof("Setting %s requests[%q] = %v", ctr, res, r)
+			glog.V(4).Infof("Setting %s requests[%q] = %v", ctr, res, r)
 		}
 		for res, cfg := range ctrcfg.Limits {
 			want := calculate(cfg, clusterSize)
 			r := resource.NewQuantity(0, guessFormat(res))
 			r.SetMilli(want)
 			newReqs[ctr].Limits[apiv1.ResourceName(res)] = *r
-			glog.V(0).Infof("Setting %s limits[%q] = %v", ctr, res, r)
+			glog.V(4).Infof("Setting %s limits[%q] = %v", ctr, res, r)
 		}
 	}
+	if reflect.DeepEqual(s.lastReqs, newReqs) {
+		return
+	}
+
+	glog.V(0).Infof("Update resource for nodes: %d, cores: %d",
+		clusterSize.Nodes, clusterSize.Cores)
+	logRequirements(newReqs)
+	s.lastReqs = newReqs
 	// Update resource target with new resources.
 	if err = s.k8sClient.UpdateResources(newReqs); err != nil {
 		glog.Errorf("Update failure: %s", err)
 	}
 }
 
+func logRequirements(reqs map[string]apiv1.ResourceRequirements) {
```

Comment:
Probably worth to log both the old and new requirements?

Reply:
First setup will also be logged, so no need to log the old as it has been logged previously.